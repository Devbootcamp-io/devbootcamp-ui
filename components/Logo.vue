<template>
  <nuxt-link to="/" class="logo">
    <img src="/icons/logo.png" />
  </nuxt-link>
</template>

<style lang="scss" scoped>
.logo {
  img {
    width: 100px;
    margin-top: 10px;
  }
}
</style>
