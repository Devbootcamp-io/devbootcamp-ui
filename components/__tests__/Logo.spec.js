import { shallowMount } from "@vue/test-utils"
import Logo from "../Logo"

const factory = () => {
  return shallowMount(Logo, {})
}

describe("Logo", () => {
  test("mounts properly", () => {
    const wrapper = factory()
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test("renders properly", () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
