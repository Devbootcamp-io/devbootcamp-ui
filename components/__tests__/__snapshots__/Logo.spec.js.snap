// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders properly 1`] = `
<div class="VueToNuxtLogo">
  <div class="Triangle Triangle--two"></div>
  <div class="Triangle Triangle--one"></div>
  <div class="Triangle Triangle--three"></div>
  <div class="Triangle Triangle--four"></div>
</div>
`;
